ui/src/app/store/machine/types.ts

```diff
@@ -249,15 +249,16 @@ export type BaseMachine = BaseNode & {
   pool: ModelRef;
   power_state: PowerState;
   power_type: string;
-  pxe_mac_vendor: string;
-  pxe_mac: string;
+  pxe_mac_vendor?: string;
+  pxe_mac?: string;
   spaces: string[];
   sriov_support: boolean;
+  status_message: string;
```

Comment:
This was also missing - it's the secondary row in the machine list's status column